response/core/models/user_external.py

```diff
@@ -11,6 +10,7 @@ class Meta:
     app_id = models.CharField(max_length=50, blank=False, null=False)
     external_id = models.CharField(max_length=50, blank=False, null=False)
     display_name = models.CharField(max_length=50, blank=False, null=False)
+    full_name = models.CharField(max_length=50, blank=True, null=True)
```

Comment:
This was added to consolidate slack profile data returning `fullname` and the e`ExternalUser` models not including this.  Having read the Slack docs, `fullname != display_name` so we can't substitute one for another.

Reply:
Interesting. Any idea how they differ? Should we consider also returning `full_name` in the API response?

Reply:
The difference seems to be this: https://get.slack.help/hc/en-gb/articles/115004692303-Change-how-names-are-displayed
We probably want `full_name` on the UI, so yeah, I'll add it to the API response.